frontend/src/components/login/Logout.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+const Logout = () => (
```

Comment:
Doesn't look reasonable login folder to contain Logout.js. I suppose the folder should be named more commonly auth or something similar.